Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/person/ui/details/items/CwaUserCard.kt

```diff
@@ -47,7 +47,9 @@ class CwaUserCard(parent: ViewGroup) :
                 .find { char -> !Character.isDigit(char) }
                 .toString()
             when (delimiter) {
-                "/" -> dateOfBirthFormatted.split("-").joinToString(delimiter)
+                "/" -> dateOfBirthFormatted.split("-").run {
+                    listOf(this[1], this[2], this[0])
```

Comment:
it is not always 3 elements , it could be 2 only. I suggest to use `getOrNull(index)` 
```kotlin
listOfNotNull(getOrNull[1], getOrNull[2], getOrNull[0])
```

Reply:
Oh yes, already forgot about that again, thanks 😅